src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -9,6 +9,8 @@
 
     <IsSourceProject>false</IsSourceProject>
     <IsPackable>true</IsPackable>
+    <!-- TODO: Enable after AvoidRestoreCycleOnSelfReference hack is removed. -->
```

Comment:
Is there an issue to remove that hack?

Reply:
Don't think so. This was introduced with https://github.com/dotnet/runtime/commit/75206ed0853b39501ae13219be6f55bfa7096c49. @ericstj do we have an issue for that?

I tried fixing this myself but that required pinning quite a lot dependent packages on preview versions which would break sourcebuild.